server.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "https://static.modelcontextprotocol.io/schemas/2025-09-16/server.schema.json",
+  "name": "io.github.kontent-ai/mcp-server",
+  "description": "Connect to Kontent.ai to manage content, types, taxonomies, and workflows via natural language",
+  "status": "active",
+  "websiteUrl": "https://kontent.ai",
+  "repository": {
+    "url": "https://github.com/kontent-ai/mcp-server",
+    "source": "github"
+  },
+  "version": "0.19.0",
```

Comment:
isn't this the old version? 😂 You wrote such a nice script and didn't use it? or is it intentionall? :D